media/js/firefox/firstrun/new-firstrun.js

```diff
@@ -9,8 +9,31 @@
         var scene = document.getElementById('scene');
         var skipbutton = document.getElementById('skip-button');
 
-        var hideSkipButton = function () {
-            skipbutton.style.display = 'none';
+        var hideOrShowSkipButton = function (data) {
+            switch(data.data.url) {
+            case 'signin':
+            case 'signup':
+            case 'reset_password':
+                skipbutton.style.display = 'block';
+                break;
+            default:
+                skipbutton.style.display = 'none';
+                break;
+            }
+        };
+
+        var enableSkipButton = function () {
+            skipbutton.style.backgroundColor = '#0996f8';
+            skipbutton.style.borderColor = '#0670cc';
+            skipbutton.style.color = '#fff';
+            skipbutton.style.opacity = '1';
+        };
+
+        var disableSkipButton = function () {
+            skipbutton.style.backgroundColor = '#ebebeb';
+            skipbutton.style.borderColor = '#b1b1b1';
+            skipbutton.style.color = '#6a6a6a';
+            skipbutton.style.opacity = '.5';
```

Comment:
To keep concerns separated, let's use CSS classes to change the appearance of the button instead of manipulating with JS.